### ISSUE TITLE
Fix filenames in .deb package

### ISSUE THREAD

**Issue by Azhrei** [RPTools/maptool#2739]
**Describe the bug**
When the Linux `.deb` package is unpacked, some of the filenames are weird.

**To Reproduce**
1. Unpack the `.deb` file on a Linux system.  The file structure will look like this for 1.8.5:
```
/opt/maptool
├── [4.0K]  bin/
│   └── [217K]  MapTool*
├── [4.0K]  lib/
│   ├── [ 16K]  app/
│   ├── [1.5M]  libapplauncher.so
│   ├── [ 262]  maptool-MapTool.desktop
│   ├── [ 251]  maptool-MapTool-MimeInfo.xml
│   ├── [ 68K]  MapTool.png
│   └── [4.0K]  runtime/
└── [4.0K]  share/
    └── [4.0K]  doc/
```
2. The `.desktop` filename should be just `MapTool.desktop`.
3. Other filenames aren't a problem, but the `.desktop` is copied to the user's desktop so it's what users see every day.

**MapTool Info**
- Version: 1.8.5 (but likely applies to all version)
- Install: New

**Desktop (please complete the following information):**
 - OS: Ubuntu (likely applies to all Debian-based distributions)
 - Version 18.04

**Comment by kayila** [comment]
I don't think this is actually a problem... The file in question gets
stored in a folder that contains .desktop files for many different
applications, but the name of the file itself is irrelevant. The important
entry (and the entry that the user will see) is the value of the Name field
contained within the .desktop file, not the .desktop file prefix.

**Comment by duckinator** [comment]
To add on to what @kayila said: `.desktop` files aren't copied to the users desktop. They're stored in system-wide directories, and are used for building lists of shortcuts.

For example, here's a picture showing some programs that show up in my Applications menu, and the files where they come from:

![Screenshot_2021-06-15_02-37-26](https://user-images.githubusercontent.com/39698/122005336-84f52400-cd83-11eb-8043-27f6eb94e288.png)

The folder containing those files is not my desktop, _and_ the name of the shortcut is not based on the filename.

**Comment by Azhrei** [comment]
Thanks for the info.  When I installed MT on Ubuntu 18.04 (a “minimal” installation), MT did not show up in the “Show Applications” menu (I didn’t check Applications).  Copying the file to the system-wide directory that you provided, does work, as does copying it to the per-user directory under ~/.local/share/applications.  It was when I copied it that I noticed the file name.  I did see that it appeared in my “Show Applications” menu by the proper name.  Just a little bit of OCD in wanting the file name to be cleaner. :)

Closing this issue.
